packages/dotcom/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { getEpicViewLog } from '../../shared/src/lib/viewLog';
+export { getWeeklyArticleHistory, incrementWeeklyArticleCount } from '../../shared/src/lib/history';
```

Comment:
Yes this is the downside to all of this - we have to be specific about what to export from shared. Otherwise it pulls in loads of unwanted code. The same happens with the types.
I spent a lot of time experimenting but do not have a better solution for this unfortunately